connectors/src/connectors/github/lib/utils.ts

```diff
@@ -0,0 +1,67 @@
+export const GITHUB_CONTENT_NODE_TYPES = [
+  "REPO_FULL",
+  "REPO_ISSUES",
+  "REPO_DISCUSSIONS",
+  "REPO_CODE",
+  "REPO_CODE_DIR",
+  "REPO_CODE_FILE",
+] as const;
+export type GithubContentNodeType = (typeof GITHUB_CONTENT_NODE_TYPES)[number];
+
+/**
+ * Gets the type of the Github content node from its internal id.
+ */
+export function matchGithubInternalIdType(internalId: string): {
```

Comment:
Can you have an explicit return type here pls 🙏

Reply:
Wait, I do already, or am I missing something? 😅

Reply:
indeed 🤦  My bad